Gems/OpenXRVk/Code/Source/OpenXRVkSwapChain.cpp

```diff
@@ -16,6 +16,33 @@
 #include <OpenXRVk/OpenXRVkUtils.h>
 #include <XR/XRFactory.h>
 
+// TODO: Expose ConvertFormat in Vulkan RHI::Reflect
+#include <../Source/RHI/Formats.inl>
+
+namespace AZ::Vulkan
+{
+    RHI::Format ConvertFormat(VkFormat format)
+    {
+#define RHIVK_VK_TO_RHI(_FormatID, _VKFormat, _Color, _Depth, _Stencil) \
```

Comment:
I assume this is used in RHIVK_EXPAND_FOR_FORMATS macro below?

Reply:
Yes